pom.xml

```diff
@@ -75,7 +75,7 @@
   </scm>
 
   <properties>
-    <maven.version>3.5.4</maven.version>
+    <maven.version>3.6.3</maven.version>
```

Comment:
We should not use `maven.version` such property is used internally by Maven, in parent we have `mavenVersion`
so please rename to `mavenVersion`

Reply:
in parent-pom 74 we still have it

Reply:
fixed